tarodb/sqlc/migrations/000002_assets.up.sql

```diff
@@ -31,6 +31,21 @@ CREATE TABLE IF NOT EXISTS genesis_points (
     anchor_tx_id INTEGER REFERENCES chain_txns(txn_id)
 );
 
+-- assets_meta is a table that holds all the metadata information for genesis
+-- assets that we either created, or bootstrapped from the relevant Base
+-- Universe.
+CREATE TABLE IF NOT EXISTS assets_meta (
+    meta_id INTEGER PRIMARY KEY,
+
+    meta_data_hash BLOB CHECK(length(meta_data_hash) = 32) UNIQUE,
```

Comment:
nit: move the `UNIQUE` keyword to before the `CHECK` keyword to match the other checks we already have in place?